config/sync/field.field.paragraph.stanford_spacer.su_spacer_size.yml

```diff
@@ -0,0 +1,23 @@
+uuid: e1e8f6c2-0706-45b3-b60e-972b168a41c7
```

Comment:
I'd normally lean toward a behavior plugin for this type of thing, but since the spacer doesn't have any other fields, having behaviors without fields would be a little weird.